src/compiler/compile.test.ts

```diff
@@ -522,9 +522,9 @@ Given a context, answer questions succintly yet complete.
   describe("includes source map when specified", async () => {
     it("returns empty source map when no identifiers", async () => {
       const prompt = `
-  Given a context, answer questions succintly yet complete.
-  <system>context</system>
-  <user>Please, help me with question!</user>
+Given a context, answer questions succintly yet complete.
+<system>context</system>
+<user>Please, help me with question!</user>
```

Comment:
Why did you remove the indent here?

Reply:
TOC and consistency with other tests. Indent thingies are tested in the specific test.